lte/gateway/python/Makefile

```diff
@@ -65,7 +65,7 @@ $(BIN)/nosetests: install_virtualenv
 	$(VIRT_ENV_PIP_INSTALL) -I nose==1.3.7
 
 $(BIN)/coverage: install_virtualenv
-	$(VIRT_ENV_PIP_INSTALL) "coverage==4.5.1"
+	$(VIRT_ENV_PIP_INSTALL) "coverage==6.1.2"
```

Comment:
do we need this specific version? without version it would use latest lib, no?

Reply:
I'm no expert in these but I recall a bunch of issues with specifying python libs. @tmdzk  what's your recommendation?

Reply:
I'd recommend to do `>=` especially for the dep in the virtual env